proto/pdpb.proto

```diff
@@ -295,3 +295,35 @@ message Error {
     optional BootstrappedError     bootstrapped = 2;
     optional StoreIsTombstoneError is_tombstone = 3;
 }
+
+service PlacementService {
+    rpc Tso(Request) returns (Response) {}
```

Comment:
Why do all rpcs share the same Request and Response?
I think it's better to be `rpc Tso(TsoRequest) returns (TsoResponse)`.